HardwareApollo.java

```diff
@@ -29,24 +29,83 @@
 
 package org.firstinspires.ftc.teamcode;
 
+import com.qualcomm.robotcore.hardware.DcMotor;
 import com.qualcomm.robotcore.hardware.HardwareMap;
+import com.qualcomm.robotcore.hardware.Servo;
+import com.qualcomm.robotcore.util.ElapsedTime;
 
-public class HardwareApollo
-{
+import static org.firstinspires.ftc.teamcode.ApolloTeleop.MID_SERVO;
+
+
+public class HardwareApollo {
     /* Public OpMode members. */
+    public DcMotor leftDriveFront = null;
+    public DcMotor rightDriveFront = null;
+    public DcMotor leftDriveBack = null;
+    public DcMotor rightDriveBack = null;
+    public Servo clawDownLeft = null;
+    public Servo clawDownRight = null;
+    public Servo clawUpLeft = null;
+    public Servo clawUpRight = null;
+    public DcMotor lift = null;
+
 
     /* local OpMode members. */
     HardwareMap hwMap = null;
+    private ElapsedTime period = new ElapsedTime();
 
     /* Constructor */
-    public HardwareApollo(){
+    public HardwareApollo() {
 
     }
 
     /* Initialize standard Hardware interfaces */
     public void init(HardwareMap ahwMap) {
         // Save reference to Hardware map
         hwMap = ahwMap;
+
+        // Define and Initialize Motors
+        leftDriveBack = hwMap.get(DcMotor.class, "left_drive_back");
```

Comment:
change left_drive_back to drive_back_left
and leftDriveBack to driveBackLeft.
accordingly to all the others